validate/src/misc.rs

```diff
@@ -0,0 +1,858 @@
+//! Contains all other input validation functions.
+//!
+//! These functions do not quite fit into a specific validation concern.
```

Comment:
I don't like this module but I don't have an alternative to provide.

Reply:
i renamed it to `request`